src/pages/HomeView/Search.js

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+
+import SearchIcon from "@material-ui/icons/Search";
+import { makeStyles } from "@material-ui/core";
+
+const useStyles = makeStyles(() => ({
```

Comment:
Could you please add this styling by creating a new Search.css. Will that work?

Reply:
Should I make a Search.module.css